nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -108,6 +108,10 @@ for o in $(cat /proc/cmdline); do
         boot.panic_on_fail|stage1panic=1)
             panicOnFail=1
             ;;
+        boot.iso=*)
```

Comment:
shouldn't this be called boot.image instead ?

Reply:
both variants are acceptable for me.